### PR TITLE
Added Jest snapshot test

### PR DESCRIPTION
**Pull request by justin0022**
`createBarChart.test.js.snap` file contains the automatically generated Jest snapshot.

It gets automatically generated on the first test run, and future test runs will compare the snapshot to this one, and fail if the snapshot does not match. 

The command to run the tests is: `docker exec -it webpack_watcher npm test`
The command to update the snapshots (if the new snapshots are correct, and the old snapshot is outdated): `docker exec -it webpack_watcher npm run update-snapshot`

### CODE REVIEW

**Comment by justin0022** [comment]
If snapshot testing for D3 looks good, I'll add snapshots for all of the D3 components.

**Comment by Adonais0** [comment]
The D3 snapshot testing looks good to me. We might want to split the testing into three parts: 

- Snapshot test for D3
- Unit test for utility functions
- Enzyme test for React components

File: assets/src/__tests__/util/math.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { average } from '../../util/math'
+
+describe('average', () => {
+  it('takes an array of numbers and calculates average', () => {
+    expect(average([1,2,3,4,5])).toEqual(3)
+  })
+})
```
Comment (Adonais0) [inline assets/src/__tests__/util/math.test.js]: It's better to create another pull request for the unit test for utility functions I think?

Comment (pushyamig) [reply assets/src/__tests__/util/math.test.js]: I think it is fine to have all test in one PR, it doesn't hurt that way.

Comment (Adonais0) [reply assets/src/__tests__/util/math.test.js]: gotcha

**Comment by justin0022** [comment]
> The D3 snapshot testing looks good to me. We might want to split the testing into three parts:
> 
> * Snapshot test for D3
> * Unit test for utility functions
> * Enzyme test for React components

That sounds good to me.

**Comment by pushyamig** [comment]
@justin0022 you can merge this PR once the conflicts are resolved. The conflicts are due to Andrews changes to the docker build got merged before this. Sorry!

**Pull request merged**